app/views/assess.scala.html

```diff
@@ -39,6 +39,7 @@ <h1 class="govuk-heading-l">@messages("bars.assess.header")</h1>
                 formGroupClasses = "form-field-group",
                 name = inputForm("input.account.sortCode").name,
                 id = inputForm("input.account.sortCode").name,
+                autocomplete = Some("off"),
```

Comment:
Do we need to turn off sort code autocomplete?  I though it was just personal details that was an issue.